llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/Tiling.cpp

```diff
@@ -395,15 +402,36 @@ void TilingInterfaceTilingPass::runOnOperation() {
           StringAttr::get(context, "tiling_repeated_indices_scatter_input"),
           StringAttr::get(context, "tiling_repeated_indices_scatter_output")));
 
-  patterns.add<TilingInterfaceTilingPattern>(
-      context, linalg::LinalgTilingOptions().setTileSizes({2, 4}),
-      IREE::LinalgExt::LinalgTransformationFilter(
-          StringAttr::get(context, "tiling_pack_input"),
-          StringAttr::get(context, "tiling_pack_output")));
-
   if (failed(applyPatternsAndFoldGreedily(funcOp, std::move(patterns)))) {
     return signalPassFailure();
   }
+
+  // TODO(hanchung): Deprecate IREE specific logic. We should move to use
```

Comment:
We don't have distribution tests for LinalgExt ops, so that won't be a concern to me. Maybe they are covered by IREE TileAndDistribute pass. For the existing tiling tests, I think we can move them to use upstream methods.

Reply:
Agreed! That would be a nice cleanup!